03/app.js

```diff
@@ -1,2 +1,12 @@
-const n = 24;
-const oddNumbers = [];
\ No newline at end of file
+const n = 51;
+const oddNumbers = [];
+fillArray();
```

Comment:
Dobrze jest pisać funkcje, która otrzymują przez parametry niezbędne dane i zwracają wynik działania.
Nie operują na danych poza funkcją czyli lepiej by było napisać:

```
const n = 51;
const oddNumbers = fillArray(n);

console.log(oddNumbers);

function fillArray (max) {
  const arr = []
  for (let i = 1; i <= max; i++) {
    if (!(i % 2 === 0)) {
      arr.push(i);
    }
  }

  return arr;
}
```

PS. Oczywiście jest to wyidealizowana forma, nie zawsze się da, albo nie zawsze się opłaca, ale trzeba się starać ;P